src/biotite/database/alphafold/download.py

```diff
@@ -0,0 +1,141 @@
+# This source code is part of the Biotite package and is distributed
+# under the 3-Clause BSD License. Please see 'LICENSE.rst' for further
+# information.
+
+__name__ = "biotite.database.rcsb"
+__author__ = "Jonathan Funk"
+__all__ = ["fetch"]
+
+import requests
+from os.path import isdir, isfile, join, getsize
+import io
+
+_standard_url = 'https://alphafold.ebi.ac.uk/files/AF-'
+
+
+def fetch(uniprot_ids, target_path=None, format='pdb', overwrite=False, verbose=False):
+    """
+    Downloads file from EBI AlphaFold database in various formats.
+
+    This function requires internet connection.
+
+    Parameters
+    ----------
+    uniprot_id: str
+        A single UniProt ID of a protein
+    target_path: str
+        Path for saving the file to. If path does not exist it will be created.
+    format: {'pdb', 'cif', 'mmcif', 'pdbx'}
+        format of file to be downloaded. Default pdb
+    overwrite : bool, optional
+        If true, existing files will be overwritten. Otherwise the
+        respective file will only be downloaded if the file does not
+        exist yet in the specified target directory or if the file is
+        empty. (Default: False)
+    verbose: bool, optional
+        If true, the function will output the download progress.
+        (Default: False)
+
+    Returns
+    -------
+    files : str or StringIO or list of (str or StringIO)
+        The file path(s) to the downloaded files.
+        If a single string (a single ID) was given in `pdb_ids`,
+        a single string is returned. If a list (or other iterable
+        object) was given, a list of strings is returned.
+        If no `target_path` was given, the file contents are stored in
+        either `StringIO` or `BytesIO` objects.
+
+    Warnings
+    --------
+    Even if you give valid input to this function, in rare cases the
+    database might return no or malformed data to you.
+    In these cases the request should be retried.
+    When the issue occurs repeatedly, the error is probably in your
+    input.
+
+    Examples
+    --------
+
+    >>> import os.path
+    >>> import requests
+    >>> file = fetch("P0DPA9", "cif", path_to_directory)
+    >>> print(os.path.basename(file))
+    P0DPA9-F1-model_v3..cif
```

Comment:
I think the double suffix`..` result from a bug in your code. However, I did not find it from visual inspection. Does this appear in the current version of your code?